package.json

```diff
@@ -21,9 +21,12 @@
     "preload-fonts": "gatsby-preload-fonts"
   },
   "dependencies": {
+    "@babel/core": "^7.12.3",

```

Comment:
Revert these changes in dependencies.

Reply:
I have to uninstall `@babel/core` for this one